### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Status

Ready for review

## Description of Changes

* Description: 
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/securedrop-docs/commit/b06decad874e37327d243d344dba13c35aca3bcf#commitcomment-89069835

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/securedrop-docs/commit/5e4fad6c31e8e1108328685fb8c8816d7772f8be

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

* Resolves #
 

* Related Issues

## Testing

*

## Release 

* 

## Checklist (Optional)

- [ ] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [ ] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/backup_and_restore.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ contact us through the `SecureDrop Support Portal`_.
           to actually be deleted. SecureDrop uses ``shred`` to
           securely erase files, which takes significantly more time
           than normal file deletion. You can monitor the progress of
-          queued deletion jobs by logging in to the *Application
+          queued deletion jobs by logging into the *Application
```
Comment (jsoref) [inline docs/backup_and_restore.rst:49]: `into` is one of the most controversial correction I have. Happy to drop.

File: docs/ossec_alerts.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ The SMTP relay that you use should support SASL authentication and SMTP
 TLS protocols TLSv1.2, TLSv1.1, and TLSv1. Most enterprise email
 solutions should be able to meet those requirements.
 
-Below are the values you must specify in to configure OSSEC correctly.
+Below are the values you must specify to configure OSSEC correctly.
```
Comment (jsoref) [inline docs/ossec_alerts.rst:52]: This is related to `into` corrections. Here I don't think `in` is valuable

File: docs/source.rst [hunk 3]
```diff
@@ -177,7 +177,7 @@ to make a submission to. The page will invite you to get started with your
 first submission or to log in. It should have a logo specific to the organization
 you are submitting to.
 
-|Source Interface with Javascript Disabled|
+|Source Interface with JavaScript Disabled|
```
Comment (jsoref) [inline docs/source.rst:180]: brand

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine.

**Review by nathandyer — CHANGES_REQUESTED** [review]
LGTM, aside from the "in to" / "into" change. Once that's ready, we should be good to merge. Thanks for taking the time to fix these spelling mistakes, this is super helpful!

File: docs/ossec_alerts.rst (hunk 2 above)
Comment (nathandyer) [reply docs/ossec_alerts.rst:52]: I agree, I believe we can omit 'in' entirely here, as you are suggesting.

File: docs/source.rst (hunk 3 above)
Comment (nathandyer) [reply docs/source.rst:180]: Definitely, we want the correct capitalization for branding here. Thanks for catching that!

**Review by nathandyer — APPROVED** [review]
This looks great! Approving / merging. Thanks again for this excellent contribution! :tada:

**Pull request merged**